src/Context.php

```diff
@@ -150,6 +154,10 @@ public function clearSymbols()
      */
     public function getSymbol($name)
     {
+        if (!is_string($name)) {
+            return null;
```

Comment:
I suggest to throw `\InvalidArgumentException` to catch this where passed `$name` is not string
If it's not a string, it's a bug